src/Stratis.Bitcoin.Features.Miner/BlockDefinition.cs

```diff
@@ -82,6 +81,7 @@ public abstract class BlockDefinition
         /// </summary>
         protected bool IncludeWitness;
 
+        // TODO: Reference parameters from this.Options instead.
         protected uint BlockMaxWeight, BlockMaxSize;
```

Comment:
@codingupastorm can't we just move these in this PR? Then we can drop the TODO :)

Reply:
Donezo